### ISSUE TITLE
Date + Time feature request

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#255]
```
Run the sql command: 
SELECT (current_date + time '01:11:05') as DTWITHTIME;

Expected:
"2010-11-16 01:11:05"

Actual:
null

H2 1.3.146
Ubuntu 10.10 x64

How important/urgent is the problem for you?
This is a nice to have feature. We use it all the time in Postgres to set up 
test data with certain times for the current date.

In your view, is this a defect or a feature request?
Feature request


```

Original issue reported on code.google.com by `sjoy...@integrityemr.com` on 16 Nov 2010 at 3:43

**Comment by GoogleCodeExporter** [comment]
```
This feature will be supported in the next release. It is already committed to 
the trunk, you can try it out if you want.
```

Original comment by `thomas.t...@gmail.com` on 19 Nov 2010 at 6:38
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in version 1.2.147.
```

Original comment by `thomas.t...@gmail.com` on 21 Nov 2010 at 5:54
- Changed state: **Fixed**
